### PR TITLE
update iOS overview

### PR DESCRIPTION
**Pull request by diarmuidmcg**

### CODE REVIEW

**Comment by diarmuidmcg** [comment]
* **#13** <a href="https://app.graphite.dev/github/pr/miamtech/mealz-documentation/13?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#12** <a href="https://app.graphite.dev/github/pr/miamtech/mealz-documentation/12?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#11** <a href="https://app.graphite.dev/github/pr/miamtech/mealz-documentation/11?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `develop`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @diarmuidmcg and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Comment by ThibaultMiam** [comment]
you may need to retack

**Review by ThibaultMiam — CHANGES_REQUESTED** [review]

File: .idea/.gitignore
[the remark this answers is not in the document]
Comment (diarmuidmcg) [reply .idea/.gitignore:1]: fait

**Comment by diarmuidmcg** [comment]
### Merge activity

* **Jun 24, 10:52 AM EDT**: @diarmuidmcg started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/miamtech/mealz-documentation/11).

**Pull request merged**